firefox-ios/Client/Frontend/Browser/Tabs/Views/TabTrayViewController.swift

```diff
@@ -378,6 +385,9 @@ class TabTrayViewController: UIViewController,
         countLabel.text = count
         segmentedControl.setImage(TabTrayPanelType.tabs.image!.overlayWith(image: countLabel),
                                   forSegmentAt: 0)
+        if isTabTrayUIExperimentsEnabled {
+            experimentSegmentControl.items[1] = "\(count) \(TabTrayPanelType.tabs.label)"
```

Comment:
I think we should add a proper translated strings for this, since I don't think we can assume in all languages the number would come first. I suggest opening a new task so we get those translations in for 139, what do you think?

Reply:
Since this is for a weekly release can we keep it as is for now and I'll create a follow up task to add for 139?

Reply:
Yeah follow up task works, anyway I don't think we could get the strings in time for the weekly